db/migrate/20230501161349_create_income_limits_tables.rb

```diff
@@ -0,0 +1,90 @@
+class CreateIncomeLimitsTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :std_counties do |t|
+      t.string :name, null: false
```

Comment:
**Question:** does the `id` field get added automatically for these tables?

Reply:
yes, the ID is automatically added as a primary key